src/Groups/group_characters.jl

```diff
@@ -3045,8 +3047,8 @@ function symmetric_parts(characters::Vector{GAPGroupClassFunction}, n::Int)
     length(characters) == 0 && return eltype(typeof(characters))[]
     tbl = parent(characters[1])
     return [class_function(tbl, chi)
-            for chi in GAPWrap.SymmetricParts(GAPTable(tbl),
-                         GAP.GapObj([chi.values for chi in characters]), n)]
+            for chi in GAPWrap.SymmetricParts(GapObj(tbl),
+                         GapObj([GapObj(chi) for chi in characters]), n)]
```

Comment:
For this kind of conversion you could now also write this:
```suggestion
                         GapObj(characters; recursive=true), n)]
```